internal/promotion/runner/builtin/schemas/git-clone-config.json

```diff
@@ -14,6 +14,28 @@
       "description": "The URL of a remote Git repository to clone. Required.",
       "minLength": 1
     },
+    "user": {
```

Comment:
Within my own head, I am debating whether it would be better to call this `author`.

Point in favor of `user`: That's what it looks like in `.gitconfig`:

```
[user]
	email = kent.rancourt@gmail.com
	signingkey = 38E66067946A0A68
	name = Kent Rancourt
```

Point in favor of `author`: Commits use `Author`:

```
commit 4bc829e4e1e01fa2757bbab198ecf4c5bc11de93 (upstream/main, main)
Author: Kent Rancourt <kent.rancourt@gmail.com>
Date:   Thu Apr 24 14:27:15 2025 -0400

    fix: follow up on patching apps with nil status (#3918)
    
    Signed-off-by: Kent Rancourt <kent.rancourt@gmail.com>
```

cc @hiddeco for any opinion he may have.

I don't feel strongly one way or the other, but felt it was worth bringing up.

Reply:
Despite the fact of `user` appearing in the config and not `author`, I can't think of any way in which these details are used _apart from_ signing and attributing authorship to a commit. This configuration does _not_, for instance, have anything to do with authentication and the ability to pull from or push to a remote.

```
[user]
	email = kent.rancourt@gmail.com
	signingkey = 38E66067946A0A68
	name = Kent Rancourt
```

I too am now leaning toward `author` (in both steps, because you're correct that we should be consistent).

Reply:
I am leaning towards `author` as well, as it is about who's authoring the commit.

Reply:
I think we're all in agreement then.